### ISSUE TITLE
RowToList?

### ISSUE THREAD

**Issue by chexxor** [paf31/purescript-foreign-generic#35]
Should we make a Decode instance for Records now that RowToList is available?

**Comment by paf31** [comment]
Yes, definitely. We'll need to anyway, since 0.12.0 will remove `Generic` support for records.
